tribe-ext-map-show-all-events-with-same-venue/src/tribe-events-ajax-maps.js

```diff
@@ -0,0 +1,711 @@
+/**
+ * @file This file contains all map view specific javascript.
+ * This file should load after all vendors and core events javascript.
+ * @version 3.0
```

Comment:
Updating this header a little to make it clear it's effectively an override of PRO's own JS file of the same name could be a nice addition. Not vital.